### PR TITLE
Change Management

### PR DESCRIPTION
**Pull request by evisochek**
## Status
In Progress

## Description
The firewall change management is a complex and delicate process that needs to be performed with extreme caution due to the consequences that can be caused due to a mistake,
If you use Pan-Os or Panorama as your enterprise firewall and Jira or ServiceNow as your enterprise ticking this pack will assist you perform a well coordinated and documented process.

### CODE REVIEW

**Comment by ShirleyDenkberg** [comment]
@evisochek I edited the README.
We still need the following:
- Links to all the graphics (I've put in place holders for these graphics)
- Link to the playbook README

**Comment by ShirleyDenkberg** [comment]
@evisochek Doc updates completed.

File: content-repo/extra-docs/packs/Change_Management.md [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+   4. In the Query section, enter a query to ensure the only change management incidents are fetched. (In the example below, *cm* is the detected project for change management.)
+
+     ![Query](../../../docs/doc_imgs/reference/Change_Management/Jira_query.PNG)
+
+11. Since it is important to document all the processes in the playbook for future reference, if you are using the Jira integration, make sure all the relevant mirroring settings are enabled (https://xsoar.pan.dev/docs/reference/integrations/jira-v2#configure-incident-mirroring).
+
+
+
+
+### ServiceNow
```
Comment (David-BMS) [inline content-repo/extra-docs/packs/Change_Management.md:89]: Maybe make sure to remind that this is for incidents and not the dedicated SNOW module

File: content-repo/extra-docs/packs/Change_Management.md [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+
+
+
+9. Load the incident JSON file and search for the values that you previously recorded and map them accordingly. For example, if the value of *Request IP SRC* was 1.1.1.1 in Jira, search for it in the JSON file and map it to the relevant field. Repeat this for all the custom fields.
+
+   ![Map_fields](../../../docs/doc_imgs/reference/Change_Management/Jira_Map_fields.PNG)
+
+
+10. Create an instance of the **Atlassian Jira v2** integration.
+   1. Select **FW change management** as the incident type.
```
Comment (David-BMS) [inline content-repo/extra-docs/packs/Change_Management.md:74]: Consider changing the numbering

**Review by David-BMS — APPROVED** [review]
Please review the comments
Other then that looks great

**Pull request merged**